lti_consumer/lti_1p3/constants.py

```diff
@@ -43,3 +43,10 @@
 ])
 
 LTI_1P3_ACCESS_TOKEN_SCOPES = []
+
+LTI_1P3_CONTEXT_TYPE_MAP = {
```

Comment:
It might be cleaner for this to be an [enum](https://docs.python.org/3/library/enum.html), especially if custom context types aren't supported. I think the preferred code style around here is to avoid having callers passing strings around that map to other constants. It's better for everybody to refer to the same constant.

So:
```
class LTI_1P3_CONTEXT_TYPE(enum.Enum):
    group = 'http://purl.imsglobal.org/vocab/lis/v2/course#CourseGroup'
    course_offering = 'http://purl.imsglobal.org/vocab/lis/v2/course#CourseOffering'
    course_section =  'http://purl.imsglobal.org/vocab/lis/v2/course#CourseSection'
    course_template = 'http://purl.imsglobal.org/vocab/lis/v2/course#CourseTemplate'
```

And you can ensure they're used like so:
```
context_claim_types = [
           context_type.value
            for context_type in context_types
            if isinstance(context_type, Enum) and context_type in LTI_1P3_CONTEXT_TYPE
        ]
```